benzene_gas.out

```diff
@@ -0,0 +1,14401 @@
+ Entering Gaussian System, Link 0=g09
```

Comment:
Are these .out files required to run the software, or are they just what you are using for your problem? If they are not required for everybody, then they either shouldn't be included, or could be included as an example problem for other people to get started with.

Reply:
The .out files are what this software package analyzes. I guess they're not technically required, but you can't really do anything with the software without them. I can move them to an example. thanks!